chpl/chpl-service/src/main/java/gov/healthit/chpl/entity/questionableActivity/QuestionableActivityDeveloperEntity.java

```diff
@@ -177,5 +180,22 @@ public DeveloperEntity getDeveloper() {
     public void setDeveloper(DeveloperEntity developer) {
         this.developer = developer;
     }
+
+    public String getReason() {
+        return reason;
+    }
+
+    public void setReason(String reason) {
+        this.reason = reason;
+    }
+
+    @Override
+    public String toString() {
```

Comment:
Love this. Can you do it for the DTO too? Unless it's already there, in which case, can you update it so we're moving to the same format for all `toString()`s

Reply:
The associated DTO is a little trickier since it extends a base QuestionableActivityDTO class.  I'll give it a try, though.

Reply:
Even if it doesn't have the elements from the base class, and just has the ones that are in the extension, it'd still be better than the default toString

Reply:
Eclipse IDE allowed my access inherited methods.  I left the default notation of using the method name "getId()=xxx" to help indicate that the value is coming from the base object and is not from the current object.